curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-more-complex-shape-using-css-and-html.english.md

```diff
@@ -92,40 +92,41 @@ tests:
 
 ```html
 <style>
-.heart {
-  position: absolute;
-  margin: auto;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-  background-color: pink;
-  height: 50px;
-  width: 50px;
-  transform: rotate(-45deg);
-}
-.heart::after {
-  background-color: pink;
-  content: "";
-  border-radius: 50%;
-  position: absolute;
-  width: 50px;
-  height: 50px;
-  top: 0px;
-  left: 25px;
-}
-.heart::before {
-  content: "";
-  background-color: pink;
-  border-radius: 50%;
-  position: absolute;
-  width: 50px;
-  height: 50px;
-  top: -25px;
-  left: 0px;
-}
+<style>
```

Comment:
```suggestion
```

Reply:
@moT01, @thecodingaviator I fixed this on behalf of user.